libindy/src/api/mod.rs

```diff
@@ -19,7 +19,12 @@ use utils::ctypes;
 
 pub type IndyHandle = i32;
 
-pub type WalletHandle = i32;
+//pub type WalletHandle = i32;
+#[derive(Debug, Hash, PartialEq, Eq, Copy, Clone)]
```

Comment:
It should be possible (and it will be very helpful and safe) to force Rust compiler to use i32 representation

Reply:
Yes, I added a repr transparent.
Rust documentation is here: https://doc.rust-lang.org/1.26.2/unstable-book/language-features/repr-transparent.html